[IMP][account_voucher_tax] Added action server to validate when from

JulioSerna opened the pull request:
bank statement reconcile payment in secondary currency sometimes
create Journal Item by Rounding adjustment to high

The Behavior cannot reproduce and is related the vauxoo issue#1460

PR DUMMY https://github.com/Vauxoo/lodigroup/pull/563

JulioSerna committed:
[IMP][account_voucher_tax] Added action server to validate when from
bank statement reconcile payment in secondary currency sometimes
create Journal Item by Rounding adjustment to high

The Behavior cannot reproduce and is related the vauxoo issue#1460

nhomar commented:
@JulioSerna 

Can you put this logic into a method and just call teh method and add a proper unittest to the method.. please.!

hbto commented on account_voucher_tax/__openerp__.py:
```diff
@@ -45,7 +45,8 @@
         "account_voucher_tax_view.xml",
-        "security/ir.model.access.csv"
+        "security/ir.model.access.csv",
+        "data/ir_actions_server.xml"
```
Please add the trailing comma at the end of the new 
element being added,
next time new change will involve only one line.

JulioSerna committed:
[IMP][account_voucher_tax] Change logic of validation from
action server to method in account_move_line and added test
to validate actions server

[IMP][account_voucher_tax] Add missing trailing comma

[IMP][account_voucher_tax] Apply pylint

JulioSerna commented:
@nhomar @hbto 

Changes applied
